android/src/main/java/com/facebook/flipper/plugins/databases/DatabasesFlipperPlugin.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018-present Facebook. All Rights Reserved.
+
+package com.facebook.flipper.plugins.databases;
+
+import android.content.Context;
+import com.facebook.flipper.core.FlipperConnection;
+import com.facebook.flipper.core.FlipperPlugin;
+import com.facebook.flipper.plugins.databases.impl.SqliteDatabaseDriver;
+import java.util.Collections;
+import java.util.List;
+
+public class DatabasesFlipperPlugin implements FlipperPlugin {
+
+    private static final String ID = "Databases";
+
+    private final DatabasesManager databasesManager;
+
+    public DatabasesFlipperPlugin(Context context) {
+        this(new SqliteDatabaseDriver(context));
+    }
+
+    public DatabasesFlipperPlugin(DatabaseDriver databaseDriver) {
+        this(Collections.singletonList(databaseDriver));
+    }
+
+    public DatabasesFlipperPlugin(List<DatabaseDriver> databaseDriverList) {
+        databasesManager = new DatabasesManager(databaseDriverList);
+    }
+
+    @Override
+    public String getId() {
+        return ID;
+    }
+
+    @Override
+    public void onConnect(FlipperConnection connection) {
+        databasesManager.setConnection(connection);
+    }
+
+    @Override
+    public void onDisconnect() {
+        databasesManager.setConnection(null);
+    }
+
+    @Override
+    public boolean runInBackground() {
+        return false;
```

Comment:
Just thinking out loud here, but it would be great if we could find a way to make this available "offline". Not sure how feasible that actually is given that databases could be quite large.